_opensearch/install/plugins.md

```diff
@@ -21,8 +21,14 @@ You can install individual plugins on an OpenSearch cluster.
 
 ## Install a plugin
 
+The install command takes a plugin id, which may be any of the following:
```

Comment:
Switch to: 


"To install a plugin, use the install plugin command by adding the plugin id, which can be any of the following:

- An official OpenSearch plugin name
- Maven coordinates to the plugin zip file
- A URL to a plugin zip file

If you're in stalling an official OpenSearch plugin, use:

```
bin/opensearch-plugin install <plugin-name>
```

For a plugin installed via zip, use:

```
bin/opensearch-plugin install <name|zip-file|url>
```